app/layout.tsx

```diff
@@ -16,7 +18,14 @@ export default function RootLayout({
 }>) {
   return (
     <html lang="en">
-      <body className={inter.className}>{children}</body>
-    </html>
+    <body>
+      <nav className="flex justify-center gap-4 p-4 bg-gray-100">
```

Comment:
Can you take a look at Shadcn [Menubar](https://ui.shadcn.com/docs/components/menubar) component and see if we can integrate them?